modules/programs/alot.nix

```diff
@@ -142,17 +151,36 @@ in
       '';
     };
 
+    extraConfigStructured = mkOption {
```

Comment:
I think you can just use the `extraConfig` name. Like in `programs.git.extraConfig`.

Renaming it to `extraConfig` would remove the possibility of having both attribute set values and string values simultaneously but as long as the attribute set variant is expressive enough I don't think that should be a problem.

Concerning the expressiveness, is this enough? It's not necessary to support attribute sets of attribute sets for example?

Reply:
Now that I have completed the PR, there shouldnt be a need for extraConfig but I like to keep both. It's an escape hatch for newer versions that could change the config, and allow a smoother transition when starting conversion of the config to nix.